### PR TITLE
chore(drive): reduce withdrawal limits [WIP]

### PR DESCRIPTION
**Pull request by shumkov**
## Checklist:

- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have added or updated relevant unit/integration/functional/e2e tests
- [ ] I have added "!" to the title and described breaking changes in the corresponding section if my code contains any
- [ ] I have made corresponding changes to the documentation if needed

**For repository code-owners and collaborators only**
- [ ] I have assigned this pull request to a milestone

## Release Notes

- **New Features**
	- Introduced a new daily withdrawal limit versioning system, allowing for enhanced withdrawal functionality.
	- Added support for a constant daily withdrawal limit of 2000 Dash.
	- Implemented a new protocol version (v6) with updated features across various components.

- **Bug Fixes**
	- Improved error handling for withdrawal transactions and asset unlock scenarios.

- **Documentation**
	- Updated comments and clarifications in the codebase for better understanding of withdrawal processes and error handling.

### CODE REVIEW

File: packages/rs-platform-version/src/version/drive_abci_versions/drive_abci_method_versions/v4.rs [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+    DriveAbciEngineMethodVersions, DriveAbciEpochMethodVersions,
+    DriveAbciFeePoolInwardsDistributionMethodVersions,
+    DriveAbciFeePoolOutwardsDistributionMethodVersions,
+    DriveAbciIdentityCreditWithdrawalMethodVersions, DriveAbciInitializationMethodVersions,
+    DriveAbciMasternodeIdentitiesUpdatesMethodVersions, DriveAbciMethodVersions,
+    DriveAbciPlatformStateStorageMethodVersions, DriveAbciProtocolUpgradeMethodVersions,
+    DriveAbciStateTransitionProcessingMethodVersions, DriveAbciVotingMethodVersions,
+};
+
+pub const DRIVE_ABCI_METHOD_VERSIONS_V4: DriveAbciMethodVersions = DriveAbciMethodVersions {
```
Comment (shumkov) [inline packages/rs-platform-version/src/version/drive_abci_versions/drive_abci_method_versions/v4.rs:14]: It must be version 5 already

File: packages/rs-platform-version/src/version/v5.rs [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+use crate::version::fee::v1::FEE_VERSION1;
+use crate::version::protocol_version::PlatformVersion;
+use crate::version::system_data_contract_versions::v1::SYSTEM_DATA_CONTRACT_VERSIONS_V1;
+use crate::version::system_limits::v1::SYSTEM_LIMITS_V1;
+use crate::version::ProtocolVersion;
+
+pub const PROTOCOL_VERSION_5: ProtocolVersion = 5;
+
+/// This version contains some fixes for withdrawals.
+pub const PLATFORM_V5: PlatformVersion = PlatformVersion {
```
Comment (shumkov) [inline packages/rs-platform-version/src/version/v5.rs:34]: It must be version 6

**Comment by thephez** [comment]
The PR should probably be named either "reduce withdrawal limitations" or "increase withdrawal limit" since the limit is going from 1000->2000. The limit is going up, while the limitations are going down. Confused yet? :zany_face:

File: packages/rs-platform-version/src/version/v5.rs (hunk 2 above)
Comment (QuantumExplorer) [reply packages/rs-platform-version/src/version/v5.rs:34]: nope

**Pull request closed without merging**